datasets/us/hi/med_exclusions/us_hi_med_exclusions.yaml

```diff
@@ -0,0 +1,43 @@
+title: US Hawaii State Medicaid Exclusions
+entry_point: crawler.py
+prefix: us-medhi
+coverage:
+  frequency: daily
+  start: "2024-10-08"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Medical providers excluded from Hawaii's Medicaid program.
+description: |
+  The State of Hawai'i Department of Human Services maintains a list of providers
+  who have been excluded from the state program.
+   > An excluded provider is an individual or entity that is not allowed to receive
+   > reimbursement for providing Medicare and Medicaid services in any capacity,
+   > even if they are not on this listing by the Med-QUEST Division.
+publisher:
+  name: Hawai'i Department of Human Services
+  description: >
+    > The Hawaii Department of Human Services is a state agency that provides
+    > essential programs and services to support the well-being of individuals and
+    > families, including healthcare, financial assistance, and social services,
+    > with a focus on improving economic security and fostering self-sufficiency.
+  acronym: DHS
+  url: https://humanservices.hawaii.gov
+  official: true
+  country: "us"
```

Comment:
where is this quote from? I can't find it anywhere.

Reply:
I asked because mentioning financial assistance is a very specific thing. If it's something we've synthesised, sure, it's not a quote, but I'm still unsure where it came from, because financial assistance isn't mentioned on their about page.

Anyway I'm making it more specific to the department.